src/Integration.Vsix.UnitTests/ErrorList/SonarErrorListDataSourceTests.cs

```diff
@@ -73,8 +74,10 @@ public void MefCtor_CheckExports()
             // Set up importers for each of the interfaces exported by the test subject
```

Comment:
Usually we do not check mef exports like this do we?

Reply:
This one is a special case because it exports more than one interface and is a singleton, so the usual test helper doesn't do what we need.
We want to check two things:
1) it exports all of the expected interfaces, and 
2) all of those exports return the same instance of the class.